tests/ai_ml/tflite2.pm

```diff
@@ -0,0 +1,44 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: This module install tensorflow2-lite and downloads
+#   a test program in python, a model, labels and an image.
+#   Then, it runs the model with the image as input.
+# Maintainer: Guillaume GARDET <guillaume@opensuse.org>
+
+use strict;
+use warnings;
+use base "opensusebasetest";
+use testapi;
+use utils;
+use Utils::Architectures 'is_aarch64';
+
+sub run {
+    my ($self) = @_;
+
+    $self->select_serial_terminal;
+    # Install required software
+    zypper_call('in tensorflow2-lite python3-Pillow');
+
+    select_console('user-console');
+    # Perform tests in a separate folder
+    script_run('mkdir tflite2_tests && pushd tflite2_tests');
```

Comment:
If this fails then tests would be run in the same folder. Why not use `assert_script_run` here as well?

Reply:
Fixed